provider/aws/formation/rack.json

```diff
@@ -1992,6 +1993,28 @@
         "Recurrence": { "Ref": "ScheduleRackScaleUp" }
       }
     },
+    "SpotScheduleRackScaleDown":{
+      "Type":"AWS::AutoScaling::ScheduledAction",
+      "Condition": "SpotInstancesAndSetScheduleRack",
+      "Properties":{
+        "AutoScalingGroupName": { "Ref": "SpotInstances" },
+        "DesiredCapacity": "0",
+        "MaxSize": "0",
+        "MinSize": "0",
+        "Recurrence": { "Ref": "ScheduleRackScaleDown" }
+      }
+    },
+    "SpotScheduleRackScaleUp":{
+      "Type":"AWS::AutoScaling::ScheduledAction",
+      "Condition": "SpotInstancesAndSetScheduleRack",
+      "Properties":{
+        "AutoScalingGroupName": { "Ref": "SpotInstances" },
+        "DesiredCapacity": "1",
+        "MaxSize":"2",
```

Comment:
Shouldn't we use the same values as the ASG declaration? Assuming that the scaleUp action is resetting the ASG to its original state.
https://github.com/convox/rack/blob/master/provider/aws/formation/rack.json#L2550

        "MinSize": "0",
        "MaxSize": "1000",